src/Layer/TiledGeometryLayer.js

```diff
@@ -0,0 +1,163 @@
+import GeometryLayer from './GeometryLayer';
+import Picking from '../Core/Picking';
+
+class TiledGeometryLayer extends GeometryLayer {
+    /**
+     * A layer extending the {@link GeometryLayer}, but with a tiling notion.
+     *
+     * @constructor
+     * @extends GeometryLayer
+     *
+     * @param {string} id - The id of the layer, that should be unique. It is
+     * not mandatory, but an error will be emitted if this layer is added a
+     * {@link View} that already has a layer going by that id.
+     * @param {THREE.Object3d} object3d - The object3d used to contain the
+     * geometry of the TiledGeometryLayer. It is usually a
+     * <code>THREE.Group</code>, but it can be anything inheriting from a
+     * <code>THREE.Object3d</code>.
+     * @param {Object} [config] - Optional configuration, all elements in it
+     * will be merged as is in the layer. For example, if the configuration
+     * contains three elements <code>name, protocol, extent</code>, these
+     * elements will be available using <code>layer.name</code> or something
+     * else depending on the property name.
+     *
+     * @throws {Error} <code>object3d</code> must be a valid
+     * <code>THREE.Object3d</code>.
+     */
+    constructor(id, object3d, config) {
+        super(id, object3d, config);
+
+        this.protocol = 'tile';
+        this.lighting = {
+            enable: false,
+            position: { x: -0.5, y: 0.0, z: 1.0 },
+        };
+    }
+
+    /**
+     * Picking method for this layer. It uses the {@link Picking#pickTilesAt}
+     * method.
+     *
+     * @param {View} view - The view instance.
+     * @param {Object} coordinates - The coordinates to pick in the view. It
+     * should have at least <code>x</code> and <code>y</code> properties.
+     * @param {number} radius - Radius of the picking circle.
+     *
+     * @return {Array} An array containing all targets picked under the
+     * specified coordinates.
+     */
+    pickObjectsAt(view, coordinates, radius) {
```

Comment:
you can declare in `class geometryLayer`
```js
 pickObjectsAt(view, coordinates, radius = this.defaultPickingRadius) {
        return Picking.pickTilesAt(view, coordinates, radius, this);
 }
```
and remove `pickObjectsAt` in `TiledGeometryLayer`

Reply:
But `TiledGeometryLayer` uses `pickTilesAt` and `GeometryLayer` uses `pickObjectsAt`.